app/src/main/java/br/com/concrete/magic/presentation/expansions/ExpansionsActivity.kt

```diff
@@ -28,21 +31,30 @@ class ExpansionsActivity : AppCompatActivity() {
 
         activityExpansionsBinding.postsRecyclerView.adapter = mAdapter
 
+        val horizontalDecoration = DividerItemDecoration(
+                activityExpansionsBinding.postsRecyclerView.getContext(),
+                DividerItemDecoration.VERTICAL)
+        val horizontalDivider = ContextCompat.getDrawable(this, R.drawable.horizontal_divider)
+        horizontalDecoration.setDrawable(horizontalDivider!!)
```

Comment:
Evite utilizar o `!!`. Caso o horizontalDivider possa ser nulo, trate usando um `let`.
```suggestion
        val horizontalDivider = ContextCompat.getDrawable(this, R.drawable.horizontal_divider)
        horizontalDivider?.let {
            horizontalDecoration.setDrawable(it)
        }
```